core/src/main/java/de/jplag/normalization/EdgeType.java

```diff
@@ -0,0 +1,29 @@
+package de.jplag.normalization;
+
+/**
+ * Enum for types of edges in normalization graph. Given two statements A and B, A comes before B, there is such an edge
+ * between A and B if...
+ */
+enum EdgeType {
+    /**
+     * B reads from a variable A writes.
```

Comment:
```suggestion
     * B reads from a variable A writes to.
```
(makes the statement clearer imo, same applies to the other comments in the class)

Reply:
It is inconsistent, but I've gone the opposite way to fix it, it's now just `B reads a variable A writes`. More concise and consistent with my thesis

Reply:
Good suggestions all around though, thank you!